### PR TITLE
XW-3631 | Handle all discussion page types (5)

### PR DESCRIPTION
**Pull request by JamesHawking**
## Links

* https://wikia-inc.atlassian.net/browse/XW-3631

## AC
Make sure we handle differences in left rail, links, and icon changes on each discussion page
Fixed width container

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: front/main/app/templates/components/design-system/local-navigation-dropdown.hbs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{{#design-system.wds-dropdown as |dropdown|}}
+	{{#dropdown.toggle class='wds-tabs__tab-label'}}
+		{{#if model.href}}
+			<a href="{{model.href}}">
+				{{#if model.image-data}}
```
Comment (hakubo) [inline front/main/app/templates/components/design-system/local-navigation-dropdown.hbs]: please use partial
https://emberjs.com/api/ember/2.14/classes/Ember.Templates.helpers/methods?anchor=#ember2455389&show=inherited,protected,private,deprecated

File: front/main/app/templates/components/design-system/local-navigation.hbs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<ul class="wds-tabs">
+	{{#each model as |firstLevelItem|}}
+		<li class="wds-tabs__tab{{if (equal firstLevelItem.href '/d/f') ' wds-is-current'}}">
+			{{#if firstLevelItem.items}}
+				{{design-system.local-navigation-dropdown model=firstLevelItem}}
+			{{else}}
+				<div class="wds-tabs__tab-label">
+					<a href="{{firstLevelItem.href}}">
```
Comment (hakubo) [inline front/main/app/templates/components/design-system/local-navigation.hbs]: this looks a lot like the example above that should use partial, maybe component?

File: front/main/app/templates/components/discussion-entities-list-layout.hbs [hunk 3]
```diff
@@ -27,69 +27,65 @@
 	redirectUrl=redirectUrl
 }}
 <div class="site-body-discussion post-list-view">
-	{{discussion-hero-unit
+	{{!discussion-hero-unit
```
Comment (hakubo) [inline front/main/app/templates/components/discussion-entities-list-layout.hbs]: remove?

File: front/main/app/templates/components/discussion-user-activity-layout.hbs [hunk 4]
```diff
@@ -12,30 +12,27 @@
 	{{/ember-wormhole}}
 {{/if}}
 <div class="site-body-discussion post-list-view">
-	{{discussion-hero-unit
+	{{!discussion-hero-unit
```
Comment (hakubo) [inline front/main/app/templates/components/discussion-user-activity-layout.hbs]: remove

File: front/main/app/templates/components/discussion-user-activity-layout.hbs [hunk 5]
```diff
@@ -12,30 +12,27 @@
...
-			{{yield}}
-		</div>
-		<div class="rail right large-3 extra-large-2 columns">
-			{{discussion-user-activity-glossary}}
+	<div class="discussion-content-wrapper">
+		<div class="discussion-page row">
+			<div class="rail left large-3 extra-large-2 columns">
+				{{left-rail-user-insights}}
+				{{discussion-side-button
+							route='discussion'
```
Comment (hakubo) [inline front/main/app/templates/components/discussion-user-activity-layout.hbs]: too many tabs

File: front/main/app/templates/components/discussion-user-entities-list-layout.hbs [hunk 6]
```diff
@@ -18,7 +18,7 @@
 }}
 
 <div class="site-body-discussion post-list-view discussion-user-list">
-	{{discussion-hero-unit
+	{{!discussion-hero-unit
```
Comment (hakubo) [inline front/main/app/templates/components/discussion-user-entities-list-layout.hbs]: remove

File: front/main/app/styles/_settings.scss [hunk 7]
```diff
@@ -83,7 +83,7 @@ $discussion-rail-title-color: #333;
 
 $discussion-desktop-bg-color: #f6f6f6;
 $discussion-desktop-post-detail-bg-color: $color-white;
-$discussion-rail-separator-color: #e6e6e6;
+$discussion-rail-separator-color: #ccc;
```
Comment (rybmat) [inline front/main/app/styles/_settings.scss]: do we have access here to wds color variables? if yes, then why not to use them?

Comment (JamesHawking) [inline front/main/app/styles/_settings.scss]: unfortunately, we don't have access wds color variables

File: front/main/app/components/left-rail-user-details.js [hunk 8]
```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
```
Comment (JamesHawking) [inline front/main/app/components/left-rail-user-details.js]: Did on line 2

File: front/main/app/components/design-system/local-navigation-dropdown-level-2.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const {Component, computed} = Ember;
+
+export default Component.extend({
+	tagName: '',
+	stickedClassName: computed('model', 'currentIndex', function () {
```
Comment (JamesHawking) [inline front/main/app/components/design-system/local-navigation-dropdown-level-2.js]: unfortunately not

File: front/main/app/styles/component/design-system/_community-header.scss [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+					$color-page-border
+	);
+
+	margin: 0 auto;
+
+	// z-index needs to be at least 5 to avoid appearing avatars from avatar_stack component (in community rail module)
+	// above the local nav dropdowns
+	z-index: 5;
+
+	// fixme
```
Comment (JamesHawking) [inline front/main/app/styles/component/design-system/_community-header.scss]: let's keep it for further work

**Pull request merged**